crates/iota-sdk-types/src/crypto/multisig.rs

```diff
@@ -53,6 +53,15 @@ pub enum MultisigMemberPublicKey {
     ZkLogin(ZkLoginPublicIdentifier),
 }
 
+impl MultisigMemberPublicKey {
+    crate::def_is_as_into_opt!(
+        Ed25519(Ed25519PublicKey),
+        Secp256k1(Secp256k1PublicKey),
+        Secp256r1(Secp256r1PublicKey),
+        ZkLogin as zklogin(ZkLoginPublicIdentifier),
```

Comment:
Just curious as to why we need the `as`?

Reply:
Without it the snake_case method becomes `as_zk_login` and so one